laws.md

```diff
@@ -60,7 +60,7 @@ New Judges are not bound by the decisions of old Judges. New Judges may, however
 
 **213.** If the rules are changed so that further play is impossible, or if the legality of a move cannot be determined with finality, or if by the Judge's best reasoning, not overruled, a move appears equally legal and illegal, then a constitutional convention should be held.
 
-**304.** PARSING VOTES: A vote must be a Pull Request (PR) comment containing only a "yay" (a vote for a proposal) or a "nay" (a vote against a proposal). Case sensitivity is irrelevant. 
```

Comment:
If it bugs you switch to rich diff and you will not see that line as changed. As the rule has not be changed in any form and you guys already barked at adding white space I keep with my settings to remove it.